Helper/Webhook.php

```diff
@@ -66,24 +65,23 @@ public function makeWebhookRequest($webhookType, $data, $klaviyoId=null)
         $err = curl_errno($curl);
 
         if ($err) {
-            $this->_klaviyoLogger->log(sprintf('Unable to send webhook to %s with data: %s', $url, json_encode($data)));
+            $this->_klaviyoLogger->log(sprintf("Unable to send webhook to $url with data: $data"));
         }
 
         // Close cURL session handle
         curl_close($curl);
+
         return $response;
     }
 
     /**
-     * @param array data
+     * @param string data
```

Comment:
Could we add a description to this method?

Reply:
Does this work?

```
* @param string $data json payload used to create hmac signature
```

Reply:
```suggestion
     * Returns an HMAC signature for webhooks
     * @param string data
```